yarn-project/p2p/src/services/libp2p/libp2p_service.ts

```diff
@@ -230,6 +234,19 @@ export class LibP2PService<T extends P2PClientType = P2PClientType.Full> extends
     const blockProposalTopic = createTopicString(TopicType.block_proposal, protocolVersion);
     const blockAttestationTopic = createTopicString(TopicType.block_attestation, protocolVersion);
 
+    const preferredPeersEnrs: ENR[] = config.preferredPeers.map(enr => ENR.decodeTxt(enr));
```

Comment:
Preferred peers are provided as 'direct' peers to gossipsub. They don't need to be discovered.